packages/hop-node/src/gasboost/GasBoostTransaction.ts

```diff
@@ -98,7 +100,7 @@ class GasBoostTransaction extends EventEmitter implements providers.TransactionR
   reorgWaitConfirmations: number = 1
   originalTxParams: providers.TransactionRequest
 
-  type?: number
+  type?: number // type 0 or type 2
```

Comment:
could we make this type stricter?

```suggestion
  type?: 0 | 2
```

Reply:
Resolved with https://github.com/hop-protocol/hop/pull/414/commits/1cb67b11caae1c4deae52a0c15ef2734bdfe58eb